examples/ct/ctmapper/mapper/mapper.go

```diff
@@ -120,11 +128,13 @@ func (m *CTMapper) oneMapperRun(ctx context.Context) (bool, error) {
 	// Fetch the current map values for those domains:
 	getReq := &trillian.GetMapLeavesRequest{
 		MapId:    m.mapID,
-		Key:      make([][]byte, 0, len(domains)),
+		Index:    make([][]byte, len(domains)),
 		Revision: -1,
 	}
-	for k := range domains {
-		getReq.Key = append(getReq.Key, []byte(k))
+	i := 0
+	for d := range domains {
+		getReq.Index[i] = HashDomain(d)
```

Comment:
Allocating the array directly is slightly more efficient than using append.
http://stackoverflow.com/questions/21362950/golang-getting-a-slice-of-keys-from-a-map

This would be a bit more elegant if domains was a slice where I could iterate over `i,d := range domains`, but it's a map, which means range iterates over `key, values` rather than `index, values`